app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] @Team Great work on your registration page, however the layout slightly differed with the given design, the button width should be equal to the input fields width, check the comparison below

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/banzaman/Blog-app/assets/61408860/2d3fa2dd-6b5b-47df-84c5-17daeb4a222b) |  ![image](https://github.com/banzaman/Blog-app/assets/61408860/b19e6f15-fd25-486e-845c-151cf010aa72)

Reply:
Let's work on that